sdk/storage/azblob/blockblob/client_test.go

```diff
@@ -646,6 +646,83 @@ func (s *BlockBlobUnrecordedTestsSuite) TestPutBlobFromURL() {
 	_require.Equal(destBuffer, sourceData)
 }
 
+func (s *BlockBlobUnrecordedTestsSuite) TestPutBlobFromURLWithCopySourceTagsReplace() {
```

Comment:
Let's add a test case for not specifying x-ms-copy-source-tags-options, the default should be replace.

Reply:
Added!